spark/src/test/scala/geotrellis/spark/io/file/cog/COGFileSpatialSpec.scala

```diff
@@ -69,6 +69,35 @@ class COGFileSpatialSpec
     }
   }
 
+  describe("COGLayerReader and readSubsetBands") {
```

Comment:
Mb to cover with the same test `S3` backend too?

Reply:
That sounds good. I'll see if I can add a test for S3 as well.